test/moduleId.html

```diff
@@ -5,6 +5,9 @@
 		<title>QUnit ModuleId Test Suite</title>
 		<link rel="stylesheet" href="../dist/qunit.css">
 		<script src="../dist/qunit.js"></script>
+		<script>
+			QUnit.warning.off();
+		</script>
```

Comment:
The first is because of https://github.com/jquery/qunit/pull/918/files#r53465639 , which should be implemented via getters (and only in environments where getters are available), and the second is because we automatically call `QUnit.reset` in the `testDone` handler, which I think we can factor out so the warning is only generated inside of a test.